### DISCUSSION TITLE
Email configuration

### DISCUSSION THREAD

**Discussion by komir2004** [akeeba/panopticon#297] [Q&A (Community Support)] [answered]
Hello, I have a problem with the email configuration in Panopticon. I can't find a way to enter the SMTP password in the settings.
Also, when I try to enable SMTP and save the changes, the button turns gray and nothing happens. 
I edited the config.php file manually and added the SMTP data, but I still don't receive any email notifications from Panopticon.
Is there something else I need to do to make it work?
I appreciate your help.

**Comment by nikosdion** [comment]
I'll publish 1.0.4 tomorrow and you'll be able to see the missing field. This version also has a send test email button.

**Comment by komir2004** [comment]
Thank you. I have updated to the new version and now the test email is working (I received the test email).
However, right after that email, I got an extension update for one site, but I did not receive any more emails, even though there are more than one page with extension updates.

**Reply by nikosdion** [reply]
Remember that the emails are queued. They are not sent immediately. The mail sending task has a lower priority to the update installation task. Once the updates are installed, you'll get your emails (about a minute after the updates are installed).

**Reply by nikosdion** [reply, marked as answer]
In versions 1.0.0 through 1.0.4 (inclusive) the emails are queued, they are not sent immediately. If you have a lot going on, [a single, or just a few, CRON jobs may not be enough](https://github.com/akeeba/panopticon/wiki/CRON-Jobs#busy-installations-will-need-more-than-one-cron-job) to get all the scheduled work done in a reasonable amount of time. Since the queued emails will only be sent when the `sendmail` scheduled tasks executes, you might have a delay of several minutes, or even hours, before the emails are sent.

Version 1.0.5 will include a switch which allows you to send emails immediately, without queuing them. Please remember that if your mail settings are wrong, your mail server is slow, or you have too many recipients you may end up with a timeout error. That's why this settings will be set to enqueue emails by default. If you feel adventurous, you can change it and see how it goes.
